packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -380,10 +408,14 @@ const styledTemplate = (opts: StyledTemplateOpts, meta: Metadata): t.Node => {
 export const compiledTemplate = (node: t.Expression, sheets: string[], meta: Metadata): t.Node => {
   const nonceAttribute = meta.state.opts.nonce ? `nonce={${meta.state.opts.nonce}}` : '';
 
+  const keyAttributeExpression = getKeyAttributeExpression(node);
+
   return template(
     `
-  <CC>
-    <CS ${nonceAttribute}>{%%cssNode%%}</CS>
+  <CC ${keyAttributeExpression ? buildKeyAttribute(keyAttributeExpression, '--cc') : ''}>
```

Comment:
You should be able to use the original `key` expression here, without appending `--cc`

Reply:
I assumed the same but React was complaining about duplicate keys. I suspect storybook may about been mounting the story twice in the same tree for some reason - I'll double check this

Reply:
Yep, that is the case - I could only remove these if I did a `yarn clean`. This could be related to https://github.com/atlassian-labs/compiled/issues/800 . I'll look into this as this is breaking the dev loop.

Reply:
Let me take a look at this as well, since I thought the new babel setup fixed it and I forgot to close out that issue